OpenSim/Simulation/Manager/Manager.cpp

```diff
@@ -752,7 +751,7 @@ bool Manager::
 integrate(SimTK::State& s)
```

Comment:
Does this deprecated function works as intended anymore? Perhaps if it's called only once? I wonder if we should remove this? Not sure.